### ISSUE TITLE
Broken intel compiler in cee-compute020 nightlies

### ISSUE THREAD

**Issue by ikalash** [sandialabs/Albany#923]
It appears the compiler on cee-compute020 is broken: https://sems-cdash-son.sandia.gov/cdash/build/46473/configure .  @jewatkins would you be able to have a look and see if the module needs to be updated?  I think you were the one who most recently switched the modules in the CEE nightlies.  It also looks like there's an mkl related issue - mkl library is not being found.

**Comment by ikalash** [comment]
Thanks @jewatkins !  They must have messed it up recently somehow.

**Comment by jewatkins** [comment]
For pnetcdf/netcdf, it looks like there's a `sems-dev-parallel-netcdf/1.12.3` but I don't see an equivalent for `sems-dev-netcdf-c/4.8.1-parallel`. Do you see one?

**Comment by ikalash** [comment]
No, there isn't one. I switched to the following and the build appears to be working.

module load sems-dev-netcdf-c/4.7.3
module load sems-dev-parallel-netcdf/1.12.3

**Comment by jewatkins** [comment]
Ah okay I just looked at that module's config and it has everything we need so it should be fine, thanks!

**Comment by ikalash** [comment]
Thanks for checking.  I'll close this tomorrow provided the build is back up.

**Comment by ikalash** [comment]
Thanks for catching this @jewatkins .  Is this an MPI/OpenMPI -related problem?  I suggest to contact CEE support to report the issue.

**Comment by jewatkins** [comment]
Did it work when you ran this manually? I think I'll have to build everything to diagnose a bit more before sending a ticket to someone. Not sure if its an issue with cee or sems.

**Comment by ikalash** [comment]
I never ran manually, just built.  I could try it if you think it's important.

**Comment by jewatkins** [comment]
Okay no problem. I think I'll try myself and see what happens, no need to run on your side.

**Comment by ikalash** [comment]
Ok, thanks @jewatkins !

**Comment by jewatkins** [comment]
@ikalash Could you try these modules in the nightlies? https://github.com/sandialabs/Albany/commit/2461872d45fd7b3c621be5677cd6cdee5857427b
I confirmed it passes an mpi test. I will let you know if Albany works but I think it should.

**Comment by ikalash** [comment]
Yes.  I just updated the scripts.  Will check tomorrow what happens with the build.  Thanks!

**Comment by jewatkins** [comment]
great, I'm currently running Albany tests and it looks like it's working.
